app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/subscription/SubscriptionFlowManager.scala

```diff
@@ -132,8 +132,6 @@ class SubscriptionFlowManager @Inject() (requestSessionData: RequestSessionData,
 
     val selectedFlow: SubscriptionFlow =
       (registrationDetails, maybeOrgType, registrationDetails.customsId, journey) match {
-        case (_: RegistrationDetailsOrganisation, Some(CdsOrganisationType.Partnership), _, _) =>
```

Comment:
Removing this case statement seems to fix the "java.lang.IllegalStateException: Incorrect Subscription flowname migration-eori-Partnership" error reported on slack.

In fact, looking at the remaining cases I think this can be further reduced to two cases - OrganisationSubscriptionFlow or IndividualSubscriptionFlow

Reply:
Those flows looks the same, so I think it's good change.

Reply:
Can you remove all unnecessary things from `SubscriptionFlows`? Because Partnership is not used anymore and it's still defined there.

Reply:
I'll have a go and see how easy it is.

Reply:
It should be used only in tests after those changes

Reply:
That is what I'm worried about!  :-)
I have a feeling that once I start hacking away at the SubscriptionFlows and Manager there will be very little left.